pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -413,32 +402,30 @@ def create_server_instance(svr_name):
                     except Exception as ex:
                         exc_type, exc_value, exc_traceback = \
                             sys.exc_info()
-                        tb = repr(traceback.format_exception(
+                        tb = traceback.format_exception(
                             exc_type,
                             exc_value,
-                            exc_traceback))
-                        raise click.ClickException('Exception failure of '
-                                                   '"--mock-server" python '
-                                                   'script %r. '
-                                                   'Exception: %r\n'
-                                                   'Traceback\n%s' %
-                                                   (file_path, ex, tb))
+                            exc_traceback)
+                        raise click.ClickException(
+                            'Exception failure of "--mock-server" python '
+                            'script {0!r}. Exception: %r\n'
+                            'Traceback\n{1}'.format(file_path, ex, tb))
```

Comment:
`tb` is a list of lines, so to display them as one would expect, it is not sufficient to remote the `repr()` but in addition, the lines need to be joined with newline.

Reply:
I can fix that in an upcoming PR that improves error messages.